src/it/scala/org/apache/spark/sql/snowflake/SFTestData.scala

```diff
@@ -12,84 +12,114 @@ trait SFTestData extends SQLTestData {
 
   import org.apache.spark.sql.test.SQLTestData.CourseSales
 
-  override protected lazy val testData: DataFrame =
-    spark.createDataFrame((1 to 100).map(i => TestData(i, i.toString)))
+  override protected lazy val testData: DataFrame = {
```

Comment:
In some new test case for Spark 3.2, it may use the `view` for the test DataFrame.
To make sure that kind of test case work, We also need to create a view after we convert a local relation to a Snowflake table.